src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -621,6 +641,8 @@ def connection_update(cmd, client,  # pylint: disable=too-many-locals
         else:
             logger.warning('client_type is not dotnet, ignore "--config-connstr"')
 
+    if linker.get('configurationInfo') and linker.get('configurationInfo').get('customizedKeys'):
```

Comment:
could be simplified like
`linker.get('configurationInfo', dict()).get('customizedKeys')`

similar below

Reply:
It can't be simplified like above because linker has a key named configurationInfo but its value is None. The suggestion code will throw exception.